text/0000-temp-config-settings.md

```diff
@@ -13,65 +13,368 @@ nit-picking the "gotchas" in code examples. :heart:
+way of managing their styles including teams within product teams. This means that
+there is no way of really measuring the implementation cost on the Pando side
+because there is currently no standadard.
+
+For the sake of the argument, let's say that the implementation cost is high.
+
+> :warning: **Note:** This is ultimately the goal to resolve via Pando. The more
+> we can standardize the way we do things, the more we can reduce the implementation
+> cost and the more we can focus on the product.
+
+### User space implementation
+
+Right now, every team uses React across the company (outside of one or two Skills
+BC's) so this is not a concern. Likewise, most teams are using CSS-in-JS so this
+enhancement should be easy to pick up and migrate. For the most part, teams should
+be reducing code by adopting and using Pando React.
+
+Panda :panda: is becoming _very_ popular across the across the industry for this exact
+reason. It is easy to pick up and use.
+
+### Impact on teaching
+
+Documentation shouldn't be too affected by this change. The only thing that would
+change is the way we teach the installation process and how to use Pando React
+via the new customizations it will offer.
+
+### Integration with other features
+
+No different here than just getting teams to use the Pando React library over
+the current chaos. Once again, this is a good thing because it will allow us to
+create a more consistent experience (both customer and developer) across the company.
+
+### Cost of migration
+
+This would introduce a breaking change **only** for teams that have yet to adopt
+Pando React. For teams that have already adopted Pando React, this would be a
+non-breaking change since the tooling is internal and it's just adding some deps
+and a config file.
+
+If we approach this via the CLI promoted above, it should be even easier for teams
+to start using Pando without doing any work.
 
 # Alternatives
 
-What other designs have been considered? What is the impact of not doing this?
+## Pando team specific
+
+The alternative is to continue to do what we are doing now which is basically
+doing what Panda already does so we can streamline the process and make it easier
+by just migrating to Panda.
+
+## Company wide
+
+The alternative is to continue to do what we are doing now which is basically
+allowing all teams to do whatever they want for FE styling which has led to a
+lot of unnecessary complexity and issues on different levels across the company.
 
 # Adoption strategy
 
-If we implement this proposal, how will existing Pando developers adopt it? Is
-this a breaking change? Can we write a codemod? Should we coordinate with
-other projects or libraries?
+This would (and should) require us to officially sunset any internal UI libary outside of
+Pando (Classic, ps-mui, and anything else). This would be a good thing because it would
+allow us to focus on one library and make it the best it can be for the company.
```

Comment:
I understand the argument that perhaps this component library _should_ encourage everyone to move off of any other library, but I don't see why that's an actual requirement here, especially not as it relates to just this implementation detail. Is that just because the Panda config would not be exposed anywhere so the tokens wouldn't be accessible anymore? If so could that not still be encapsulated into its own package?

Also perhaps worth noting that this does feel like a 180 degree turn away from the original idea of the original `tva` idea (which tbf I agree that centralizing into a single UI library is a much better long-term approach, but I still think that exposing the building blocks individually is useful, even if only during a transitionary period).

Reply:
For sure and thanks for that clarification @josh-degraw. 😸